app/assets/stylesheets/pages/_all.scss

```diff
@@ -0,0 +1,3 @@
+@import "home";
+@import "machines";
+@import "roasts";
```

Comment:
Is this importing the _home.scss, _machines.scss, and _roasts.scss files in the same directory? If so... Is this the best directory structure or best place for this file? 

The location of this file or its name may not be the most clear for what it does. It would seem to imply that all pages displayed on the website should require each of the referenced files - I don't think this is the case?

scss is not my domain, so you guys tell me.